<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CTaskSettingsGuiComp</class>
 <widget class="QWidget" name="CTaskSettingsGuiComp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>234</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Task Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="1,1,1,1,1,1000">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QCheckBox" name="ActiveCheck">
     <property name="text">
      <string>Active</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Task Name</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="TaskNameEdit"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>User-ID</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="TaskUserIdEdit"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="inputIDLabel">
     <property name="text">
      <string>Input-ID</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QStackedWidget" name="InputPageStack">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QComboBox" name="InputSelector"/>
       </item>
       <item>
        <widget class="QToolButton" name="ShowAcquisitionManagerButton">
         <property name="text">
          <string>Edit</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/Icons/Edit</normaloff>:/Icons/Edit</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLineEdit" name="InputIdEdit"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Description</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="TaskDescriptionEdit"/>
   </item>
   <item row="5" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>ActiveCheck</tabstop>
  <tabstop>TaskNameEdit</tabstop>
  <tabstop>TaskUserIdEdit</tabstop>
  <tabstop>InputIdEdit</tabstop>
  <tabstop>TaskDescriptionEdit</tabstop>
 </tabstops>
 <resources>
  <include location="../../../ImtCore/Include/imtgui/imtgui.qrc"/>
 </resources>
 <connections/>
</ui>
